src/functions.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Lunanimous\Rpc;
+
+if (!function_exists('toNim')) {
+    /**
+     * Converts from luna to nim.
+     *
+     * @param int $luna the value in luna (smallest unit)
+     *
+     * @return float the value in nim
+     */
+    function toNim($luna)
+    {
+        return $luna / 100000;
+    }
+}
+
+if (!function_exists('toLuna')) {
+    /**
+     * Converts from nim to luna.
+     *
+     * @param float $nim the value in nim
+     *
+     * @return int the value in luna (smallest unit)
+     */
+    function toLuna($nim)
+    {
+        return $nim * 100000;
```

Comment:
The official JS implementation in [`Policy.js`](https://github.com/nimiq/core-js/blob/3e6cb39625a278dcd9f4406b9ed5dff55b17b36d/src/main/generic/consensus/Policy.js#L8) also rounds here, to be sure that the result contains no decimals. You should either also round, or cast the output to `(int)` to make sure.